W-15784039 asyncapi duke (#322)
mulesoft/docs-code-builder: modules/ROOT/pages/_partials/acb-reusable-steps.adoc

. Click the image:anypoint-code-builder::icon-munit-testing.png["",18,18] (Testing) icon to open the Testing panel.
// end::munit-open-testing-panel[]
//

// ASYNCAPI BETA ACCESS//
//
// tag::asyncapi-beta-access[]

> Use of AsyncAPI implementation features requires your organization administrator to accept beta terms and conditions from Anypoint Code Builder in Anypoint Platform: 

Author: ```suggestion
Your organization administrator must accept the terms and conditions for beta AsyncAPI features: 
```
